tests/cases.py

```diff
@@ -1551,3 +1562,67 @@ def setUp(self) -> None:
             triples_factory=self.factory,
             **self.model_kwargs,
         )
+
+
+class CleanerTestCase(GenericTestCase[Cleaner]):
+    """Test cases for cleaner."""
+
+    def post_instantiation_hook(self) -> None:
+        """Prepare triples."""
+        self.dataset = Nations()
+        self.all_entities = set(range(self.dataset.num_entities))
+        self.mapped_triples = self.dataset.training.mapped_triples
+        # unfavourable split to ensure that cleanup is necessary
+        self.reference, self.other = torch.split(
+            self.mapped_triples,
+            split_size_or_sections=[24, 1592 - 24],
```

Comment:
can you use a constant instead of 1592? This is the number of triples in nations right? I think you could use `self.dataset.num_triples`?

Reply:
b8f1096